cygnus-common/spec/SPECS/cygnus-common.spec

```
@@ -171,6 +171,8 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

```suggestion
* Fri Sep 04 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

(Not sure but in older entries we did that way...)